package.json

```diff
@@ -47,6 +47,7 @@
     "eslint-plugin-import": "^2.31.0",
     "lint-staged": "15.5.0",
     "lockfile-lint-api": "^5.9.1",
+    "playwright": "1.15.1",
```

Comment:
We don't want this as a direct dependency.  It should be installed via the `npm install browser-deps` script and not required for developing locally.  That was the original idea of having that separate script.

Reply:
Have removed